spec/filter_form_spec.rb

```diff
@@ -49,11 +49,11 @@
       before do
```

Comment:
Tests are failing because Author model no longer has `last_seen_at` column.

```ruby
(byebug) Author
Author(id: integer, name: string, last_name: string, birthday: date, created_at: datetime, updated_at: datetime)
```

I'll rewrite the test soon.